README.md

```diff
@@ -100,6 +100,21 @@ docker run -p 8080:80 -d -e SKYWALKING_URL=127.0.0.1:1234,127.0.0.1:1235 rocketb
 
 The default UI address is `http://localhost:8080`.
 
+### Setting the timezone for docker image
+
+Suppose you want to use Shanghai First copy the proper zone to localtime
+
+```dockerfile
+ls /usr/share/zoneinfo
+cp /usr/share/zoneinfo/Asia/Shanghai /etc/localtime
+```
+
+Now specify your timezone
+
+```
+echo "Asia/Shanghai" >  /etc/timezone
+```
```

Comment:
And needs some description here, such as `Those dockerfile is just for developer review or refer methods.` etc.